data/processed/hebci/gbp_business_contract_research.yaml

```diff
@@ -0,0 +1,28 @@
+api_doc_url: https://www.hesa.ac.uk/support/definitions/hebci
+api_type: FETCH
+data_date: 20202002
+description_short: University income from contract research with businesses
+description: Contract research income with businesses for universities in the NUTS2 region in the academic year starting in year 
+endpoint_url: https://www.hesa.ac.uk/data-and-analysis/providers/business-community/table-2a.csv
+framework_group: knowledge_exchange
+is_experimental: False
+order: [year, nuts_id, nuts_year_spec, value.id]
+region:
+  type: NutsRegion
+  level: 2
+  source_url: https://opendata.arcgis.com/datasets/48b6b85bb7ea43699ee85f4ecd12fd36_0.zip?outSR=%7B%22latestWkid%22%3A27700%2C%22wkid%22%3A27700%7D 
+schema:
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    description: Contract research income with businesses for universities in the NUTS2 region in the academic year starting in year 
+    id: gbp_business_contract_research
+    label: Contract research income with businesses
+    type: GBP
```

Comment:
same here, we need to add a `format` (in this case these are like `int`s so it'd be `d`) or we specify `data_type: int` on top of the `type` (because `GBP.data_type: int|float`)